tests/core/contracts/conftest.py

```diff
@@ -1008,7 +1008,7 @@ def invoke_contract(api_call_desig='call',
                     func_args=[],
                     func_kwargs={},
                     tx_params={}):
-    allowable_call_desig = ['call', 'transact', 'estimateGas', 'buildTransaction']
+    allowable_call_desig = ['call', 'transact', 'estimate_gas', 'build_transaction']
```

Comment:
I didn't actually look to see, but do we need to keep 'estimateGas' and `buildTransaction` here until `estimateGas` and `buildTransaction` are removed?

Reply:
This one is ok to change since I changed the names of all the methods in the test themselves. This is the list of methods that four methods that come below this method.